apps/openmw/mwbase/mechanicsmanager.hpp

```diff
@@ -6,6 +6,8 @@
 #include <list>
 #include <stdint.h>
 
+#include "../mwworld/ptr.hpp"
```

Comment:
This include is not necessary. There is already a forward declaration for MWWorld::Ptr in mechanicsmanager.hpp.